go-controller/pkg/util/node_annotations.go

```diff
@@ -312,6 +315,38 @@ func NodeChassisIDAnnotationChanged(oldNode, newNode *kapi.Node) bool {
 	return oldNode.Annotations[ovnNodeChassisID] != newNode.Annotations[ovnNodeChassisID]
 }
 
+type ManagementPortDetails struct {
+	PfId int `json:"PfId"`
+	VfId int `json:"VfId"`
+}
+
+func SetNodeManagementPort(nodeAnnotator kube.Annotator, PfId int, VfId int) error {
```

Comment:
`SetNodeManagementPortAnnotation`

Reply:
Do you want to change all the functions in this file to include "Annotation" in the function name? For example, we already have `SetNodeManagementPortMACAddress`. Should we change that to `SetNodeManagementPortMACAddressAnnotation` too?

Reply:
Frankly all functions within this file which says "SetNode..." should have "..Annotation" at the end, because this is what they are really doing. For now, adjust only `SetNodeManagementPort`, because that's a lot of changes to add them into separate commit.